networks/local/localnode/Dockerfile

```diff
@@ -14,3 +14,4 @@ STOPSIGNAL SIGTERM
 
 COPY wrapper.sh /usr/bin/wrapper.sh
 COPY config-template.toml /etc/tendermint/config-template.toml
+COPY ../../../build/tendermint /tendermint/tendermint
```

Comment:
why do you think we need this?

Reply:
Not needed, it's just to debug why the tendermint executable is not in the mounted volume on CircleCI.

Reply:
just removed it, the experiment failed, I had to switch to `machine` in CircleCI